_data/projects/movie-py-cli.yml

```diff
@@ -0,0 +1,25 @@
++name: movie-py-cli
+ +desc: "A simple script for fetching movie details right from the terminal"
+ +site: 
+ +tags:
+ +# Note: these are tags categorizing the project, not issue labels.
+ +- movies
+ +- imdb-movies
+ +- imdb-webscrapping
+ +- imdb
+ +upforgrabs:
+ +  name: "Help Wanted"
+ +  link: https://github.com/lucifer069/movie-py-cli/labels/help%20wanted
+
++--- 
+ +desc: "A simple script for fetching movie details right from the terminal"
```

Comment:
This looks like the file contents has been duplicated, and contains some `+` characters that are unnecessary.